class_configs/pal_class_config.lua

```diff
@@ -655,32 +655,41 @@ return {
             {
                 name = "WaveHeal",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
```

Comment:
active_cond is meant to describe when something is active so this is probably wrong as it is true if the AA is ready not if it is currently on you.

Reply:
Understood. Removed the active_cond